prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
```

Comment:
Why? You can use JSX, so using `"` is preferred here

Reply:
I have no preference here. I think `singleQuote` was used more often in code before, hence I set it `true`.

Reply:
Updated.

Reply:
Unfortunately there's too many errors hard to fix with `singleQuote: false`, thus I'm reverting the change.

Reply:
We can return to this latest, let's integrate prettier firstly